sql_r_tutorial.Rmd

```diff
@@ -0,0 +1,192 @@
+---
+output:
+  html_document: default
+  pdf_document: default
+---
+# Using SQL with R
+
+Samir Char, Christodoulos Constantinides
+
+
+```{r, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+knitr::opts_chunk$set(message = FALSE)
+knitr::opts_chunk$set(warning = FALSE)
+```
+# Installing required packages
```

Comment:
Please change all the subsequent single hashtag # header lines to double hashtags ##. Ideally there should be no additional single hashtag headers in the .Rmd file.